parent/pom.xml

```diff
@@ -807,6 +807,14 @@
         <version>${project.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>io.camunda</groupId>
+        <artifactId>camunda-service</artifactId>
+        <version>${project.version}</version>
+        <type>test-jar</type>
+        <scope>test</scope>
+      </dependency>
+
```

Comment:
❓ is this needed because of where the ITs are placed?

Reply:
It's needed because `CamundaUserDetailsService` test is located in `authentication` module but it needs access to `StubbedCamundaSearchClient`. but now I'm thinking I can make a mock of UserService and get rid of this dep, let me try.